R/tunable.R

```diff
@@ -16,8 +16,6 @@ tunable.step <- function(x, ...) {
   no_param
 }
 
-step_type <- function(.step) class(.step)[class(.step) != "step"][1]
```

Comment:
Is this relevant here?

https://github.com/tidymodels/recipes/blob/8f5afe1/tests/testthat/test_skipping.R#L44

Reply:
Yes, we can remove that line as well, good catch!